gtsfm/pose_slam/pose_slam.py

```diff
@@ -20,6 +23,100 @@
 
 
 class PoseSlam:
+    """Initializes using PoseSLAM given relative pose priors."""
+
+    @staticmethod
+    def angle(R1, R2):
+        """Calculate angle between two rotations, in degrees."""
+        return np.degrees(np.linalg.norm(R1.logmap(R2)))
```

Comment:
Nit: we already have a method for this (in utils/geometry_comparisons.py I think).

Reply:
done